Include FQDN before non-Docker Hub image

cjeffers opened the pull request:
I ran into this issue while following the tutorial. The `docker pull` command looks at the image name, and if it does not start with an FQDN, it assumes the image lives on Docker Hub. In order for App Service to be able to pull from ACR, you need to change the name of the image from `mydockerimage` to `<azure-container-registry-name>.azurecr.io/mydockerimage`.

azurecla commented:
Hi __@gangsir__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
MSFT storage PM.

cjeffers committed:
Include FQDN before non-Docker Hub image

I ran into this issue while following the tutorial. The `docker pull` command looks at the image name, and if it does not start with an FQDN, it assumes the image lives on Docker Hub. In order for App Service to be able to pull from ACR, you need to change the name of the image from `mydockerimage` to `<azure-container-registry-name>.azurecr.io/mydockerimage`.
